kubernetes/linux/setup.sh

```diff
@@ -40,10 +40,10 @@ sudo apt-get install jq=1.5+dfsg-2 -y
 #used to setcaps for ruby process to read /proc/env
 sudo apt-get install libcap2-bin -y
 
-wget https://dl.influxdata.com/telegraf/releases/telegraf-1.20.3_linux_$ARCH.tar.gz
-tar -zxvf telegraf-1.20.3_linux_$ARCH.tar.gz
+wget https://dl.influxdata.com/telegraf/releases/telegraf-1.22.2_linux_$ARCH.tar.gz
+tar -zxvf telegraf-1.22.2_linux_$ARCH.tar.gz
```

Comment:
Amol, did you test all telegraf metrics with this change ?

Reply:
doing that today. will tag you once I have the results

Reply:
@vishiy please review the notes here: https://msazure.visualstudio.com/InfrastructureInsights/_sprints/backlog/Containers/InfrastructureInsights/Copper/May%202022?workitem=14284466 which describe my tests. Let me know if I missed anything that needs to be covered